sunkit_pyvista/tests/test_plotting.py

```diff
@@ -48,6 +48,12 @@ def test_plot_map_with_functionality(aia171_test_map, plotter, verify_cache_imag
                           obstime=aia171_test_map.date)
     plotter.plot_coordinates(coordinate, color='blue')
 
+    filepath = (tmp_path / "save_data.vtm")
+    plotter.save(filepath=filepath)
+
+    plotter = SunpyPlotter()
+    plotter.load(filepath)
```

Comment:
Why did the figure change?

Reply:
Ah, there might be another issue with the way we're saving plots.
`clip_interval` isn't being saved, the figure changed because the previous figure had `clim` passed to add_mesh but after loading we're not passing the previously passed `clim`, how would we handle this?

Reply:
We either save all the plot settings to a json file or we don't save them and just clarify in the documentation that these settings are not saved.

If we did the first one, how many plot settings do we need to save?

Reply:
I mean if we're passing kwargs to `add_mesh` I think we can't determine the various plot settings that has to be saved. But we could consider the useful functions we've created like clip interval? But this is the only one I can think of as of now, I think we could mention that these settings aren't saved in the documentation

Reply:
There is no reason we couldn't save the kwargs?

Reply:
We'd have to do this for every mesh then right?

Reply:
Yeah. 

I think this is worth an issue and a conversation on there. This is too big to decide on this PR.

Reply:
Alright, I'll open one then